torchvision/prototype/transforms/functional/_augment.py

```diff
@@ -17,15 +17,21 @@ def erase_image_pil(
     return to_pil_image(output, mode=image.mode)
 
 
+def erase_video(
```

Comment:
The new video kernels are trivial for now, since they only wrap the image kernels. This works as long as the image kernel supports arbitrary batch sizes, since for all the current transformations videos can be regarded as a batch of images. 

However, there are some kernels that currently don't support arbitrary batch sizes:

https://github.com/pytorch/vision/blob/9cece405db3a8301463b219d9e0885142837ae4f/test/prototype_transforms_kernel_infos.py#L906-L907

We need to fix them in order for the video kernel to work properly.

In the future, if we have transforms that work on the temporal dimension, i.e. `video.shape[-4]`, we actually need to implement explicit video kernels.

Reply:
For those that dont support arbitrary dimensions, you are able to reshape the T dimension into N, send to the Image kernel and then reshape back as a temporarily workaround.

Reply:
I've added #6670. How urgent is it for this to work for videos but not for images? If not as urgent, I would fix the image kernels and benefit from that here.

Reply:
Not urgent, we can do it on a follow up.